Reflections/Week_3_Reflection.md

```diff
@@ -0,0 +1,9 @@
+**Will Pratt**
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+* When traveling between rooms I have have addressed most bugs that could come from incorrect user input. I have added a way that healing items can be used and worked on the logic for enemy skills so that they have cooldowns as well. I also created a story class that will handle the intro and story elements like outputting the title and class selection.
```

Comment:
Great progress for this week! It might be about time to consider letting others test and play the game as you add more rooms and content.